scripts/apps/authoring/views/authoring-header.html

```diff
@@ -74,7 +74,7 @@
             <div class="authoring-header__item"
                  sd-validation-error="error.slugline"
                  data-required="schema.slugline.required"
-                 ng-if="schema.slugline"
+                 ng-if="schema.slugline && editor['slugline'].section === 'header'"
```

Comment:
imo would be better like `&& editor['slugline'].section !== 'content'` to be backward compatible